src/clj/test/cards/resources.clj

```diff
@@ -1095,6 +1095,34 @@
           ;; the fact that we got this far means the bug is fixed
           (is (= 1 (count (get-hardware state))) "Spy Camera installed"))))))
 
+(deftest officer-frank
+  ;; Officer Frank - meat damage to trash 2 from HQ
+  (do-game
+      (new-game (default-corp [(qty "Swordsman" 1) (qty "Hedge Fund" 2)])
+                (default-runner [(qty "Officer Frank" 1) (qty "Skulljack" 1) (qty "Respirocytes" 4)]))
+   (play-from-hand state :corp "Swordsman" "Archives")
+   (take-credits state :corp)
+   (core/move state :runner (find-card "Respirocytes" (:deck (get-runner))) :hand)
+   (play-from-hand state :runner "Officer Frank")
```

Comment:
Unless I missed something, this line will fail on occasion when the random starting hand does not contain Officer Frank, since your deck has 6 cards. You should either adjust the deck to only have 5 cards, or use the `(starting-hand)` utility to forcefully set the runner's starting hand to include the cards you need.